### ISSUE TITLE
 ls --help  is not available on Mac OS:   make it more visible by bold text

### ISSUE THREAD

**Issue by dhanyapushpadas** [swcarpentry/shell-novice#1264]
[02-filedir.md](https://github.com/swcarpentry/shell-novice/edit/gh-pages/_episodes/02-filedir.md)
line 190 and line 196 , make the text in parentheses bold so that it is more visible to learners that` ls --help `is not available on macOS and `man ls`  is not available in Git Bash

**Comment by mszarko** [comment]
In a recent Carpentries workshop I observed, this section still generated a lot of confusion for the Mac users (they either didn't see the bolded section, or didn't understand it). Could a note be added to explain what users on the respective operating systems should do instead? For example, line 189 could read something like:
 
1. We can pass a `--help` option to the command (not available on macOS; use man -ls instead), such as:

I definitely understand (and appreciate!) the effort to keep the script as simple as possible to avoid constant updating for every operating system. But I did make a note of this one for my own separate teaching notes, since it obviously still caused some confusion for several workshop participants.
